src/account/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+from rest_framework import generics
 
-# Create your views here.
+
+from common.mixins import RelativeURLFieldMixin
+from . import models
+from . import serializers
+
+
+class ProfileViewSet(
+    RelativeURLFieldMixin,
+    generics.ListAPIView,
+    generics.UpdateAPIView,
```

Comment:
You can use this https://www.django-rest-framework.org/api-guide/generic-views/#retrieveupdateapiview.